app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsFragment.kt

```diff
@@ -80,7 +79,7 @@ class AddTitleDescriptionsFragment : Fragment() {
         super.onCreateView(inflater, container, savedInstanceState)
 
         // TODO: add funnel?
-        source = arguments?.getInt(EXTRA_SOURCE, EDIT_TASKS_TITLE_DESC_SOURCE)!!
+        source = (arguments?.getSerializable(EXTRA_SOURCE) as InvokeSource?)!!
```

Comment:
can be simplified to `source = arguments?.getSerializable(EXTRA_SOURCE) as InvokeSource`

Reply:
Done

Reply:
Done